modules/runtime/nuxeo-runtime-stream/src/main/java/org/nuxeo/runtime/stream/StreamServiceImpl.java

```diff
@@ -256,4 +263,16 @@ public void beforeStop(ComponentManager mgr, boolean isStandby) {
             Framework.getRuntime().getComponentManager().removeListener(this);
         }
     }
+
+    protected boolean isProcessingDisabled() {
+        if (isStreamProcessingDisabled == null) {
+            if ("false".equals(Framework.getProperty(STREAM_PROCESSING_ENABLED))) {
```

Comment:
`if (Framework.isBooleanPropertyFalse(STREAM_PROCESSING_ENABLED))`

Reply:
if you want